scripts/globals/battlefield.lua

```diff
@@ -959,10 +983,10 @@ function Battlefield:onBattlefieldEnter(player, battlefield)
         self.requiredItems.wearMessage and
         player:hasItem(self.requiredItems[1])
     then
-        local itemId = self.requiredItems[1]
-        local uses = player:incrementItemWear(itemId)
-        -- Gets the total number of item uses for the given item. Default to one since that is the majority of them.
-        local totalUses = xi.battlefield.itemUses[itemId] or 1
+        local itemId    = self.requiredItems[1]
+        local uses      = player:incrementItemWear(itemId)
+        local totalUses = xi.battlefield.itemUses[itemId] or 1 -- Gets the total number of item uses for the given item. Default to one since that is the majority of them.
```

Comment:
this makes the line 163 chars wide, which was hopefully why they put the comment above instead of inline. I do the same thing when they get to wide (as defined by making me right/left scroll side-by-side diff)